src/com/belsoft/project/CustomDictionary.java

```diff
@@ -0,0 +1,64 @@
+package com.belsoft.project;
+
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CustomDictionary {
+
+    private final Map<String, String> names = new HashMap<>();
+    private final Map<String, String> ports = new HashMap<>();
```

Comment:
We intend to store port (key) and application information (value), so I would expect to have a way to model a Application (one solution: create Application class, with relevant fields)